apps/web/src/pages/ApplicantDashboardPage/components/ReviewTalentNominationDialog.tsx

```diff
@@ -258,6 +309,131 @@ const ReviewTalentNominationDialog = ({
                   intl.formatMessage(commonMessages.notProvided)}
               </FieldDisplay>
             </div>
+            {talentNomination.nominateForAdvancement && (
+              <>
+                <Separator decorative data-h2-margin="base(0)" />
```

Comment:
If you';d like, we have a `space="none"` prop that removes the margin.

Reply:
Sure, done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/13525/commits/cb17d90dc6279dd152a78feb308fc16202606a22